### PR TITLE
Mimir Buff

### PR DESCRIPTION
**Pull request by Sebbe9123**
## About The Pull Request
This PR Aims to make Mimir mk 1 and Mimir mk 2 better at what they are supposed to do.
Previously it seems nobody quite knew what Mimir did besides make you immune to xeno gas clouds.
Mimir mark 1 chest and head armor values are as follows.
Bio goes from 0 to 15.
rads dont matter
acid goes from 0 to 15

Mimir mark 2 chest and helmet armor are as follows.
Bio goes from 20 to 40
rads still don't matter
acid goes from 20 to 30.

## Why It's Good For The Game
It's hard to find information on what equipment gets used or not besides from hear-say, but Mimir is supposedly the least used armor mod thing thanks to a few different reasons with the main one apparently being that Tyr was better even without the gas immunity. 
This PR makes Mimir much more competitive with other modules and makes it excel at what it's supposed to do which just makes sense.

## Changelog
:cl:
balance: Rebalances Mimir Bio and Acid armor.
/:cl:

### CODE REVIEW

**Comment by Hekzder** [comment]
While I fully think the Mimir needs a buff to stay competitive with Tyr, I think this is a bit far potentially. This is a pretty hefty buff to both Mk2 and Mk1. Ultimately I think just Mk2 needs the most love, Mk1 is mostly fine as is IMO.

**Comment by vilereaver41** [comment]
Big buff between mk1 and mk2

**Comment by SplinterGP** [comment]
its funny considering the Mk1 has no armor and it also does nothing, as opposed to what the desc says

**Review by BraveMole — CHANGES_REQUESTED** [review]
Where is bio armor even used? For neurotoxin initial damage on projectile?

Also, update MK1 description

**Comment by Sebbe9123** [comment]
> Where is bio armor even used? For neurotoxin initial damage on projectile?
> 
> Also, update MK1 description

Did the description for both.

During the discussion before I made the PR, Kuro informed me that Bio armor is resistance to neuro spits.

**Comment by BraveMole** [comment]
> During the discussion before I made the PR, Kuro informed me that Bio armor is resistance to neuro spits.

Yeah but what are you resisting to, damage, amount of neurotox reduced? What is the effect?

**Review by BraveMole — APPROVED** [review]

**Comment by AndroidMK2** [comment]
I think the helmet mounted mimir also grants some protection. You should check them so it's the set of those 2 is not too op

File: code/modules/modular_armor/helmets.dm [hunk 1]
```diff
@@ -42,10 +42,10 @@
 
 /obj/item/helmet_module/attachable/mimir_environment_protection
 	name = "Mark 2 Mimir Environmental Helmet System"
-	desc = "Designed for mounting on a Jaeger Helmet. When attached, this system provides substantial resistance to environmental hazards, such as gases, acidic elements, and radiological exposure. Best paired with the Mimir Environmental Resistance System. Will impact mobility when attached."
+	desc = "Designed for mounting on a Jaeger Helmet. When attached, this system provides substantial resistance to environmental hazards, such as gases, biological and radiological exposure. Thhis newer model provides a large amount of protection to acid. Best paired with the Mimir Environmental Resistance System. Will impact mobility when attached."
```
Comment (Hekzder) [inline code/modules/modular_armor/helmets.dm:45]:
```suggestion
	desc = "Designed for mounting on a Jaeger Helmet. When attached, this system provides substantial resistance to environmental hazards, such as gases, biological and radiological exposure. This newer model provides a large amount of protection to acid. Best paired with the Mimir Environmental Resistance System. Will impact mobility when attached."
```

Typo

**Pull request merged**